core/src/main/resources/lib/hudson/project/configurable_tr.properties

```diff
@@ -20,7 +20,7 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-delete={0} sil
+delete={0}yi Sil
```

Comment:
```suggestion
delete={0} yi Sil
```

Reply:
Same as the others. The resulting word should be "Proje**yi**" not "Proje yi"